source/common/tracing/zipkin/span_buffer.cc

```diff
@@ -2,7 +2,7 @@
 
 namespace Zipkin {
 
-bool SpanBuffer::addSpan(SpanPtr&& span) {
+bool SpanBuffer::addSpan(Span&& span) {
```

Comment:
To make it clear that we are currently copying here, please make this method take a ```const Span&``` instead. I would also add a TODO that this needs to be looked at for perf.